lib/mojito.js

```diff
@@ -23,88 +26,48 @@ Mojito = (function () {
 
     // The main test object
     var Test = (function () {
-        var cookiePrefix = '_mojito',
-            p;
-
         var constructor = function (options) {
+            var p;
+
             this.options = Utils.extend({
                 id: null,
-                name: null,
+                name: options.id,
```

Comment:
We can default the name to the test's ID if it doesn't exist. I don't see why it's necessary to throw an error for this when some users may not need a test name variable exposed.